imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -16,9 +16,9 @@ const taxCalc = {};
 
 taxCalc.getPackageData = function () {
   const pkgData = Packages.findOne({
```

Comment:
Is this the correct pattern here?

Reply:
I think we want to use `Reaction.getPackageSettings`?

Reply:
What is supposed to be the correct pattern? @aaronjudd

Reply:
you should use the Reaction.getPackageSettings method:
https://github.com/reactioncommerce/reaction/blob/marketplace/server/api/core/core.js#L432

Reply:
Also, is this change to Avalara required to make TaxCloud work? Something seems fishy?

Reply:
No it was to make Avalara work, and I have made the correct pattern change in my latest commit, @spencern @aaronjudd @zenweasel